CHANGELOG.md

```diff
@@ -50,7 +50,7 @@
 * [BUGFIX] Improved handling of complete blocks in localblocks processor after enabling flusing [#3805](https://github.com/grafana/tempo/pull/3805) (@mapno)
 * [BUGFIX] Handle out of boundaries spans kinds [#3861](https://github.com/grafana/tempo/pull/3861) (@javiermolinar)
 * [BUGFIX] Maintain previous tenant blocklist on tenant errors [#3860](https://github.com/grafana/tempo/pull/3860) (@zalegrala)
-
+* [BUGFIX] Remove unused properties from the WAL configuration [#3911](https://github.com/grafana/tempo/pull/3911) (@javiermolianr)
```

Comment:
we need to list this as a breaking change. we do that by typing `**Breaking Change**` on the changelog line like this:

https://github.com/grafana/tempo/blob/d0f0a5fddb7b3234e70f4bd39bb592a8e0b5df50/CHANGELOG.md?plain=1#L3

that will help us find and note the removal of these config optoins for our users when we cut 2.6

Reply:
Done, I didn't think this was a breaking change because the configuration was not honored ever :P

Reply:
That is a good point, but we are removing config options that previously were parsed. It might break someone's config I suppose.